src/component/DefaultLegendContent.tsx

```diff
@@ -3,6 +3,7 @@
  */
 import React, { PureComponent, ReactNode, MouseEvent, ReactText, ReactElement } from 'react';
 import classNames from 'classnames';
+import _ from 'lodash';
```

Comment:
nit nit, me thinking aloud: I know we use `lodash` in this lib everywhere, but I wonder if there is any risk in just `typeof entry.value === 'function'` or `entry.value instanceof Function`

Reply:
@ckifer I don't see any risk in it and usually prefer native JS operations as well! I noticed we used `_.isFunction()` far more often than the native operation throughout the repo and just went with this in the name of consistency. _(lodash used 75 times vs native operation used 12 times)_

I can make that quick change if we'd prefer native JS readability over consistency with lodash's method here.

Reply:
It's all good, I just was mostly making an observation for myself. Eventually we may want to move away from lodash but the consistency is perfect for now